### PR TITLE
Deprecate jQuery UI accordion

### PR DESCRIPTION
**Pull request by orangejenny**
The last one was in case management for forms in advanced modules.

@gcapalbo / @snopoke 

Before
screen shot 2016-02-23 at 9 35 44 pm

After
screen shot 2016-02-23 at 9 36 28 pm

### CODE REVIEW

**Comment by snopoke** [comment]
Wow, cool. Having plaid with this do you think the collapsing is necessary at all?

**Comment by orangejenny** [comment]
I think the collapsing is nice if you have a lot of actions. Might actually be better to start with everything collapsed, so you can see how many actions there are, and then let you expand as needed. Thoughts?

**Comment by snopoke** [comment]
Yea, that was my thinking originally. Fine to stick with it as is or to
start with them all collapsed (if there are more than 1).

**Comment by czue** [comment]
:fire:

**Comment by orangejenny** [comment]
@snopoke sounds good, added a commit

**Comment by snopoke** [comment]
:+1: once build passes unless you think it needs more QA

**Comment by orangejenny** [comment]
I think it's good to go; I poked at it for a while getting it to work. (and admittedly, I don't want it to get stuck in QA for a while, their backlog looks non-trivial)

File: corehq/apps/app_manager/static/app_manager/js/case-config-ui-advanced.js [hunk 1]
```diff
@@ -152,16 +152,19 @@ var AdvancedCase = (function () {
...
+                var selector = "#case-" + t + "-accordion";
+
+                // Make sure all parents are set correctly so panels behave like an accordion
+                $(selector + ' > .panel > .panel-collapse').collapse({
+                    parent: selector,
+                    toggle: false,
+                });
+
+                // Hide all panels, then show the requested one
+                $(selector + ' .panel-collapse.in').collapse('hide')
```
Comment (dimagimon) [inline corehq/apps/app_manager/static/app_manager/js/case-config-ui-advanced.js]: Missing semicolon.

**Pull request merged**